requirements/ci-wheel.txt

```diff
@@ -12,13 +12,15 @@ pytest==5.2.1
 pytest-cov==2.8.1
 pytest-mock==1.11.1
 tox==3.14.0
-trustme==0.5.2; platform_machine=="x86_64"  # cryptography doesn't work on i686
-cryptography==2.8; platform_machine=="x86_64"  # cryptography doesn't work on i686
 twine==1.15.0
 yarl==1.3.0
 
 # Using PEP 508 env markers to control dependency on runtimes:
-aiodns==2.0.0; platform_system!="Windows"  # required c-ares will not build on windows
+
+# required c-ares will not build on windows and has build problems on Macos Python<3.7
+aiodns==2.0.0; sys_platform=="linux" or sys_platform=="darvin" and python_version>="3.7"
```

Comment:
@asvetlov dar**w**in is spelled with "W"
```suggestion
aiodns==2.0.0; sys_platform=="linux" or sys_platform=="darwin" and python_version>="3.7"
```

Reply:
oops